glibc.yaml

```diff
@@ -48,8 +48,8 @@ environment:
   # glibc manages FORTIFY_SOURCE on per-file basis
   environment:
     CPPFLAGS: ""
-    # https://github.com/chainguard-dev/internal-dev/issues/7756
-    GCC_SPEC_FILE: /dev/null
+    # See https://github.com/wolfi-dev/os/pull/53351#issuecomment-2963753802
+    GCC_SPEC_FILE: /home/build/openssf.spec
 
```

Comment:
if we are confident that gcc-15 fixed it, should we drop this hunk altogether and use the default build?